### ISSUE TITLE
[Feature Request] Use template variables inside docker compose

### ISSUE THREAD

**Issue by wallzero** [SelfhostedPro/Yacht#463]
**Is your feature request related to a problem? Please describe.**
I would like to use Yacht template variables inside a docker compose.

**Describe the solution you'd like**

For example:

```yaml
version: "3.8"
services:
    app:
        image: something/app
        container_name: app
        environment:
            - PUID=${!PUID}
            - PGID=${!PUID}
            - TZ=${!TZ}
        volumes:
            - ${!config}:/config
        ports:
            - 1234:1234
        restart: unless-stopped
```

**Describe alternatives you've considered**
If we can't use the template variable format `!VARIABLE` then another option would be to point containers to an environment file such as `env_file: .env`.

After inspecting inside the container, I found template variables are stored inside `/api/settings.py`. Would it be possible to also export these variables to an adjacent `/api/settings.env`?

**Additional context**
Basically I'd like to test drive containers with Yacht templates but eventually migrate to a versioned docker compose. Reusing the Yacht template variables would ease that migration.

**Comment by TheFlyingBadger** [comment]
The template variables are actually stored in the "template_variables" (i see what you did there!) table in the database. The values in the settings.py are just defaults. you could dump them out of the DB into a .env file easily enough with some py-jiggery (or whatever) if you wanted to
